src/orders/views/OrderDetails/OrderDetailsMessages.tsx

```diff
@@ -337,6 +345,22 @@ export const OrderDetailsMessages: React.FC<OrderDetailsMessages> = ({
     }
   };
 
+  const handleTransactionAction = (
+    data: OrderTransactionRequestActionMutation,
+  ) => {
+    const {
+      transactionRequestAction: { errors },
+    } = data;
+    const isError = !!errors.length;
```

Comment:
```suggestion
    const isError = !!data?.transactionRequestAction?.errors?.length;
```

Reply:
We need the `errors` later in the function